README.md

```diff
@@ -147,3 +147,7 @@ approval of the
 The source code for the DEX server and client are being developed according to
 the specification. This undertaking was approved via a second DEX
 [development proposal](https://proposals.decred.org/proposals/417607aaedff2942ff3701cdb4eff76637eca4ed7f7ba816e5c0bd2e971602e1).
+
+## FAQS
+
+- How can I integrate new assets? To add new assets, follow the instructions [here](https://github.com/decred/dcrdex/blob/master/spec/fundamentals.mediawiki/#adding-new-assets) and see existing implementations [here](https://github.com/decred/dcrdex/tree/master/server/asset).
```

Comment:
Pointing to the server/asset package isn't really right.  We need to point to a collection of commits that does it all, client, dex, and server packages.  I'll do this after DGB.

Reply:
> We need to point to a collection of commits that does it all, client, dex, and server packages. I'll do this after DGB.

Okay. I was also thinking If there could be other FAQs that need a detailed explanation, but none that I am aware of except this.